### ISSUE TITLE
I have an issue with the image loading. Images available but not displayed!!! Repository is stored in Gitlab. Site is deployed on Netlify. This bug was described before. Was it fixed?

### ISSUE THREAD

**Issue by VasilHamulyak** [netlify/netlify-cms#2277]
**Describe the bug**

> netlify-cms.js:24 TypeError: Cannot read property 'id' of undefined
    at t.default.getMediaDisplayURL (netlify-cms.js:302)
    at getLargeMediaClient.then.o (netlify-cms.js:302)

**To Reproduce**

**Expected behavior**

**Screenshots**
![Screenshot from 2019-04-09 14-19-50](https://user-images.githubusercontent.com/30534198/55797576-7a6cdf00-5ad5-11e9-9a70-b141fa3ca71c.png)

**Applicable Versions:**

 - Netlify CMS version: [e.g. 2.0.4]
 - Git provider: [Gitlab]
 - OS: [Linux]

 - Node.JS version:

**CMS configuration**
```
backend:
  name: git-gateway
  branch: master

media_folder: "src/images"
public_folder: "images"

collections:
  - name: "services"
    label: "Services"
    folder: "src/content/services"
    create: true
    fields:
      - { name: "title", label: "Title", widget: "string" }
      - { name: "order", label: "Order", widget: "number" }
      - { name: "number", label: "Number", widget: "string" }
      - { name: "image", label: "ServiceImage", widget: "image"}
      - { name: "body", label: "Body", widget: "markdown" }

```
**Additional context**

**Comment by tomrutgers** [comment]
The fix hasn't been released yet, it will come out in `2.9.1`. In the meantime you can use `2.9.1-beta.2`.

**Comment by VasilHamulyak** [comment]
Thanks)
